### ISSUE TITLE
ClusterControl volume file problem with docker

### ISSUE THREAD

**Issue by enzoveltri** [severalnines/docker#23]
I started the container using your example :

```
$ docker run --name clustercontrol \
-h clustercontrol \
-p 5000:80 \
-p 5001:443 \
-v /storage/clustercontrol/cmon.d:/etc/cmon.d \
-v /storage/clustercontrol/datadir:/var/lib/mysql \
-v /storage/clustercontrol/sshkey:/root/.ssh \
-v /storage/clustercontrol/cmonlib:/var/lib/cmon \
-v /storage/clustercontrol/backups:/root/backups \
severalnines/clustercontrol
```
it stops on "Generating key for s9s CLI" and never starts the container correctly.

If I remove the line related to cmonlib I'm able to run the container, but when I stop the container it won't start.

**Comment by ashraf-s9s** [comment]
HI @enzoveltri 

I am not able to reproduce this. Did you see this line in the docker log?
`Grant user 'dba' succeeded.`

I need to see the output of `docker logs -f clustercontrol` to get some ideas on what was happening. If you could provide it, that would be helpful.

Anyway, I've published a new image v1.7.2. Could you try with this image instead?

**Comment by ashraf-s9s** [comment]
Hi @enzoveltri 

I managed to reproduce this in the latest image. It was due to CMON HA code refactoring which happened to miss out `/etc/cmon.d/cmon.cnf` as the default path by cmon process. 

The new image contains the latest fix and is now ready. Do pull it again. I'm going to close this issue.
